www_src/components/modal/modal.less

```diff
@@ -0,0 +1,70 @@
+#modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(0,0,0,0.3);
```

Comment:
(that's what the image modal background is -- although probably a good idea to save this as `overlay` or something in colors.less maybe?)